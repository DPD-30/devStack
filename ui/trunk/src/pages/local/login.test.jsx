import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import { mount } from 'enzyme'

import PagesLocalLogin from 'pages/local/login.jsx'

jest.unmock('pages/local/login.jsx')


describe('PagesLocalLogin', () => {
    it('should match snapshot authenticated', () => {
        const test = mount(
            <Router>
                <PagesLocalLogin />
            </Router>
        )
        const comp = test.find(PagesLocalLogin)
    
        comp.setState({ isAuthenticatedLocal: true })

        expect(comp).toMatchSnapshot()
    })

    it('should match snapshot not authenticated', () => {
        const test = mount(
            <Router>
                <PagesLocalLogin />
            </Router>
        )
        const comp = test.find(PagesLocalLogin)
    
        comp.setState({ isAuthenticatedLocal: false })

        expect(comp).toMatchSnapshot()
    })
})
