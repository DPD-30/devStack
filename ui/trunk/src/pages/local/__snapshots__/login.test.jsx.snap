// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesLocalLogin should match snapshot authenticated 1`] = `ReactWrapper {}`;

exports[`PagesLocalLogin should match snapshot not authenticated 1`] = `ReactWrapper {}`;
